test/goss.yaml

```diff
@@ -64,8 +75,9 @@ file:
     owner: artemis
     group: artemis
     filetype: file
-{{ if (.Env.ACTIVEMQ_ARTEMIS_VERSION | regexMatch "(1\\.[^0-2]\\.[0-9]+|2\\.[0-9]\\.[0-9]+)") }}
     contains:
+      - brokerConfig.acceptorArtemisArgs
```

Comment:
This as I said before sounds to me as a different feature. I might be wrong thow, please let me know what you think.

Reply:
it's related, I updated README.md to add more details.